### PR TITLE
params/bootnodes: remove default etherum mainnnet bootnodes

### PR DESCRIPTION
**Pull request by huyngopt1994**
Replace the default Ethereum foundation mainnet bootnodes by Ronin bootnodes for avoiding unused discovery and peering in Network.  We have 2 options for extending the bootnodes now 
1. Overwrite by setting Bootnodes ENV
2. We automatically fetch 5 endpoints from DNS discovery, implemented in https://github.com/axieinfinity/ronin/pull/370

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -1000,10 +1000,12 @@ func setNodeUserIdent(ctx *cli.Context, cfg *node.Config) {
 // setBootstrapNodes creates a list of bootstrap nodes from the command line
 // flags, reverting to pre-configured ones if none have been specified.
 func setBootstrapNodes(ctx *cli.Context, cfg *p2p.Config) {
-	urls := params.MainnetBootnodes
+	urls := params.EtherumMainnetBootnodes
 	switch {
 	case ctx.GlobalIsSet(BootnodesFlag.Name):
 		urls = SplitAndTrim(ctx.GlobalString(BootnodesFlag.Name))
+	case ctx.GlobalBool(MainnetFlag.Name):
```
Comment (minh-bq) [inline cmd/utils/flags.go:1007]: We never set this flag when running, so this looks incorrect. The default must be `RoninMainnetBootnodes` and in this case when mainnet flag is set `urls = params.EtherumMainnetBootnodes`

Comment (huyngopt1994) [reply cmd/utils/flags.go:1007]: Yes, checked this flag, it's used for Etherum Mainnet. I have the otherway for creating flag for Ronin but it requires more flag and updating in entrypoint.sh, so let go with this suggestion.

**Comment by DNK90** [comment]
I think we should remove all ethereum network. Just add 2 types: our saigon testnet and mainnet.
It would prevent misconfiguration from users

File: mobile/params.go [hunk 2]
```diff
@@ -71,8 +71,8 @@ func GoerliGenesis() string {
 // FoundationBootnodes returns the enode URLs of the P2P bootstrap nodes operated
 // by the foundation running the V5 discovery protocol.
 func FoundationBootnodes() *Enodes {
-	nodes := &Enodes{nodes: make([]*enode.Node, len(params.MainnetBootnodes))}
-	for i, url := range params.MainnetBootnodes {
+	nodes := &Enodes{nodes: make([]*enode.Node, len(params.EtherumMainnetBootnodes))}
```
Comment (DNK90) [inline mobile/params.go:74]: I don't know if we can use it in the future or not. But this should be change to Ronin bootnodes instead, shouldn't it?

**Comment by huyngopt1994** [comment]
> I think we should remove all ethereum network. Just add 2 types: our saigon testnet and mainnet. It would prevent misconfiguration from users

Yes let me separate in to the other PR for this stuff.

File: mobile/params.go (hunk 2 above)
Comment (huyngopt1994) [reply mobile/params.go:74]: It's backward compatibilities in running testing right now, so we will keep it first.

**Review by minh-bq — APPROVED** [review]

**Pull request merged**